GroupManager/Controllers/GroupsController.cs

```diff
@@ -84,8 +84,9 @@ public async Task<ActionResult> Index()
         private async Task<string> GetGraphAccessToken(string userId, string[] scopes)
         {
             IConfidentialClientApplication cc = MsalAppBuilder.BuildConfidentialClientApplication();
+            IAccount userAccount = await cc.GetAccountAsync(ClaimsPrincipal.Current.GetMsalAccountId());
 
-            AuthenticationResult result = await cc.AcquireTokenSilent(scopes, ClaimsPrincipal.Current.ToIAccount()).ExecuteAsync(); 
+            AuthenticationResult result = await cc.AcquireTokenSilent(scopes, userAccount).ExecuteAsync(); 
```

Comment:
>AcquireTokenSilent [](start = 51, length = 18)

enhancement:Clicking on groups after signing in and providing consent should lead back to the home page, otherwise the user sees the "Connect your tenant" button again.

Reply:
@kalyankrishna1  I think this happens not because of the redirect URL, but because the cached permissions. Since you have just added a new grant, you need to sign out and sign in again to see its effect. Isn`t it?

Reply:
Added redirect to SignOut and on the HTML file, added a an explanation that the user will be logged out to refresh the cache.